### PR TITLE
Add invite accept

### PR DESCRIPTION
**Pull request by alexgraham-da**
Adds an invitation acceptance screen to investor, issuer and exchange roles.

### CODE REVIEW

File: ui/src/components/Investor/InviteAcceptScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            <div className='invite-accept-is-public'>
+                <FormToggle
+                    defaultChecked
+                    className='invite-accept-form-item'
+                    onLabel='Public'
+                    onInfo='All parties will be aware of this investor.'
+                    offLabel='Private'
+                    offInfo='Only a set of parties will be aware of this investor.'
+                    onClick={val => setIsPublic(val)}/>
+            </div>
```
Comment (alexmatson-da) [inline ui/src/components/Investor/InviteAcceptScreen.tsx:72]: We had decided previously that we will default `isPublic: true` for the investor for now. We felt the implications could be confusing to end users, so we'll add it as an option when we find the right way to present it

**Review by alexmatson-da — COMMENTED** [review]
This is looking great! I have a few suggestions outlined below

**Comment by alexgraham-da** [comment]
> This is looking great! I have a few suggestions outlined below

Great, thanks for the feedback! I incorporated all of your comments, including removing the `disabled` prop from the other Nav components.

File: ui/src/components/Exchange/Exchange.tsx [hunk 2]
```diff
@@ -25,25 +28,16 @@ const Exchange: React.FC<Props> = ({ onLogout }) => {
...
-            // accept the invite as soon as it's seen
-            const { ExchangeInvitation_Accept } = ExchangeInvitation;
-            const key = wrapDamlTuple([operator, user]);
-            ledger.exerciseByKey(ExchangeInvitation_Accept, key, {})
-            .catch(err => console.error(err))
-        }
-    }, [exchangeInvite, ledger, operator, user]);
+    const registeredExchange = useStreamQuery(RegisteredExchange);
 
     const sideNav = <ExchangeSideNav disabled={!exchangeContract} url={url}/>;
```
[the remark this answers is not in the document]
Comment (alexgraham-da) [reply ui/src/components/Exchange/Exchange.tsx]: This has been updated in the Exchange, Investor, and Issuer side navs. Was definitely a very nice simplification 😄

File: ui/src/components/Exchange/InviteAcceptScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+import { InviteAcceptTile, InviteAcceptButton, InviteTextField } from '../common/InviteAcceptTile'
```
[the remark this answers is not in the document]
Comment (alexgraham-da) [reply ui/src/components/Exchange/InviteAcceptScreen.tsx]: This should be fixed in all of the files I touched. Let me know if you spot any others.

File: ui/src/components/Investor/InviteAcceptScreen.tsx (hunk 1 above)
Comment (alexgraham-da) [reply ui/src/components/Investor/InviteAcceptScreen.tsx]: Removed and set automatically to true.

**Review by alexmatson-da — APPROVED** [review]
Nice!

**Review by dimitri-da — APPROVED** [review]

**Pull request merged**